src/rapids_metadata/__init__.py

```diff
@@ -168,10 +168,13 @@
 
 all_metadata.versions["24.12"] = deepcopy(all_metadata.versions["24.10"])
 
+# fmt: off
 del all_metadata.versions["24.12"].repositories["cugraph"].packages["cugraph-dgl"]
+del all_metadata.versions["24.12"].repositories["cugraph"].packages["cugraph-equivariant"]
 del all_metadata.versions["24.12"].repositories["cugraph"].packages["cugraph-pyg"]
 del all_metadata.versions["24.12"].repositories["cugraph"].packages["nx-cugraph"]
 del all_metadata.versions["24.12"].repositories["wholegraph"]
+# fmt: on
```

Comment:
These `fmt:` comments prevent `black` from wrapping the line with `cugraph-equivariant`. I think it's much easier to read this without the wrapping.